taxi/forms.py

```diff
@@ -0,0 +1,50 @@
+from django import forms
+from django.contrib.auth.forms import UserCreationForm
+from django.core.validators import RegexValidator
+
+from taxi.models import Driver, Car
+
+
+validate_license_number = RegexValidator(
+    regex=r"^[A-Z]{3}\d{5}$",
+    message="The license number must consist of 3 capital letters"
+            " followed by 5 numbers."
+)
+
+
+class LicenseNumberValidationForm(forms.ModelForm):
+    def clean_license_number(self):
+        license_number = self.cleaned_data["license_number"]
+        validate_license_number(license_number)
+        return license_number
+
+
+class DriverCreationForm(UserCreationForm, LicenseNumberValidationForm):
```

Comment:
you can do more simple here. Instead of creating a separate class for validation define a field here:
`license_number = forms.CharField(validators=[...`